core/templates/pages/topics-and-skills-dashboard-page/skills-list/skills-list.directive.html

```diff
@@ -70,24 +90,69 @@
 <style>
   skills-list .list-view-item {
     background: #fff;
-    margin: 20px 7.5px 0 12.5%;
-    padding: 10px 20px;
-    width: 75%;
+    box-shadow: none;
+    margin-top: 50px;
+  }
+
+  skills-list .list-item {
+    height: 140px;
+    text-align: left;
+    vertical-align: baseline;
+  }
+
+  skills-list .dashboard-table-index-column {
+    width: 5%;
+  }
+  skills-list .dashboard-table-description-column {
+    width: 25%;
+  }
+  skills-list .dashboard-table-examples-column {
+    width: 10%;
+  }
+  skills-list .dashboard-table-misconception-column {
+    width: 10%;
+  }
+  skills-list .dashboard-table-skill-column {
+    width: 15%;
+  }
+  skills-list .dashboard-table-status-column {
+    width: 20%;
+  }
+
+  skills-list .list-item td {
+    padding-top: 15px;
+  }
+
+  skills-list .skill-description {
+    font-size: 17px;
+    font-weight: bold;
+  }
+
+  skills-list .skill-topic {
+    font-size: 15px;
+    margin: 4px 0;
+    text-transform: uppercase;
+  }
+
+  skills-list .skill-topic i {
+    color: #D57B00;
   }
 
   skills-list .dashboard-table {
-    display: block;
     font-size: 15px;
-    max-height: 700px;
-    overflow-y: auto;
     width: 100%;
   }
 
   skills-list .dashboard-table .dashboard-table-headings {
     border-bottom: 2px solid #bbb;
+    text-align: center;
     padding-bottom: 0.7em;
   }
-
+  skills-list #dashboard-table-heading-skills-description {
```

Comment:
We don't use ids for CSS -- it's too hard to ensure that there's always just one on a page. Please stick to using classes instead.

Reply:
Done.